http-server-jetty/src/test/groovy/io/micronaut/servlet/jetty/JettyNotFoundSpec.groovy

```diff
@@ -62,11 +62,11 @@ class JettyNotFoundSpec extends Specification {
     @Client('/not-found')
     static interface InventoryClient {
 
-        @Get(value = '/mono/{isbn}', processes = MediaType.TEXT_PLAIN)
+        @Get(value = '/mono/{isbn}', processes = MediaType.APPLICATION_JSON)
         @SingleResult
         Publisher<Boolean> mono(String isbn)
 
-        @Get(value = '/flux/{isbn}', processes = MediaType.TEXT_PLAIN)
+        @Get(value = '/flux/{isbn}', processes = MediaType.APPLICATION_JSON)
```

Comment:
why were these changes required?

Reply:
can't deserialize boolean from text/plain

Reply:
why?

Reply:
because it has no specified structure. we could parse it as json, but that could be wrong

Reply:
shouldn't it go through the normal conversion infrastructure for `text/plain`? We already have a `String` -> `Boolean` converter

Reply:
this is what TextPlainCodec did, toString for serialization and conversion for deserialization. but going through conversion kind of defeats the point of the message handlers.

On the write side, there is this workaround: https://github.com/micronaut-projects/micronaut-core/blob/37874c634202233f35b7c9376a5edfd5d49861f2/http/src/main/java/io/micronaut/http/body/DynamicMessageBodyWriter.java#L64-L69 – it's restricted to java.lang types, and just calls toString. but this does not work on the read side since there is no real inverse of toString, only conversion. it's in breaks.adoc here: https://github.com/micronaut-projects/micronaut-core/blob/37874c634202233f35b7c9376a5edfd5d49861f2/src/main/docs/guide/appendix/breaks.adoc#L186

Reply:
I have created a test  in the TCK which verifies the issue. I only happens for Declarative HTTP Client which uses the netty implementation. I think we should support this.

Reply:
I have added back the txt plain media type and annotated the test with `@PendingFeature`